Cargo.toml

```diff
@@ -48,7 +48,7 @@ deno_bench_util = { version = "0.112.0", path = "./bench_util" }
 test_util = { path = "./test_util" }
 deno_lockfile = "0.17.1"
 deno_media_type = { version = "0.1.1", features = ["module_specifier"] }
-deno_npm = "0.15.0"
+deno_npm = "0.15.1"
```

Comment:
I did a breaking change in a patch release so that I didn't need to upgrade all our other crates.